src/_base/harness/attributes/docker-base.yml

```diff
@@ -167,6 +175,12 @@ attributes:
           enabled: false
         mysql:
           enabled: false
+        php-base:
+          environment:
+            TIDEWAYS_ENABLED: "= @('php.ext-tideways.enable') ? 'true' : 'false'"
```

Comment:
maybe this but in pipeline.base.services.php-base.environment:
```
            TIDEWAYS_ENABLED: "{{ .Values.service.tideways | default .Values.docker.services.tideways.enabled }}"
```
the deployment template templates these, and quotes, which even turns booleans to quoted string

Reply:
Updated :+1: